.buildkite/longruns/pipeline.yml

```diff
@@ -1,5 +1,5 @@
 env:
```

Comment:
Do we have to change any of the parameters for runs in this file?

Reply:
It's better to be consistent with the Julia version we'll be using for this and the CI. The longruns are broken now, but we should bring them back in the next few weeks.